demo/simple2.sysl

```diff
@@ -25,7 +25,7 @@ Server:
 
 # This is an example of apps with types made up of different apps
 MegaDatabase:
-    @package = "ServerPackage"
+    @package = "MegaDatabase"
```

Comment:
This was meant to be an example of how different applications can be in the same package